### DISCUSSION TITLE
Merci !

### DISCUSSION THREAD

**Discussion by breizhim** [marian-m12l/studio#272] [General]
Je suis impressioné par la qualité de l'outil. Ca aurait pu être juste une bidouille en ligne de commande, mais même pas ! Un vrai framework ergonomique à souhait et fiable.

J'ai pu rajouter quelques histoires trouvées ci et là, et ça fait le bonheur de mes filles.

Bravo. Et un grand, grand merci.

**Comment by fparrel** [comment]
Je me joins à ces remerciements (je ne voulais pas ouvrir de nouveau sujet pour ne pas polluer le forum).
Impressionné par le boulot de reverse engineering.
Lancé le script .sh et il marche direct sur Ubuntu sans aucune modification, génial.
Maintenant plus de limites de temps dans les histoires, meilleure qualité et j'ai en plus découvert que l'écran de la Lunii v2 est en couleurs!

**Comment by sferlisi** [comment]
Tout pareil ! Merci mille fois !

**Comment by Sekhy2** [comment]
Je me joins aussi pour les remerciements. Vraiment bien foutu, bravo. :)

**Comment by VincentSinel** [comment]
Je profite moi aussi pour vous remercier pour ce formidable travail !
J'ai découvert votre outils il y a un mois et je me suis bien amusé à m'enregistrer ainsi que ma compagne en train de lire les histoires préférés de mon fils. Il est trop au top ! Petit plus sur la création d'une playlist de musique aussi (plus besoin de mette en boucle les chansons depuis notre téléphone en voiture).
Merci merci merci !
